cmd/injector/main.go

```diff
@@ -30,26 +30,30 @@ import (
 	"github.com/dapr/kit/logger"
 )
 
-var log = logger.NewLogger("dapr.injector")
+var (
+	log  = logger.NewLogger("dapr.injector")
+	opts = options.New()
+)
 
-func main() {
-	log.Infof("starting Dapr Sidecar Injector -- version %s -- commit %s", buildinfo.Version(), buildinfo.Commit())
+func init() {
+	// Apply options to all loggers
```

Comment:
I think this should still go at the beginning of `main` maybe?

Reply:
agreed an `init` function is unnecessary here.

Reply:
Yes, I "reverted" the code to its previous state with `init`, but we can remove it.